src/components/WorkOrder/Appointment/__snapshots__/PropertyDetails.test.js.snap

```diff
@@ -22,28 +22,22 @@ exports[`PropertyDetails component should render properly 1`] = `
         >
           Tenure: Secure
         </li>
-        <li
-          class="bg-orange"
-        >
-          Address Alert: Location Alert 1 (
-          <strong>
-            type1
-          </strong>
-          )
-        </li>
-        <li
-          class="bg-orange"
+        <span
+          class="govuk-error-message lbh-error-message"
+          data-error-id=""
+          data-testid="error-message"
```

Comment:
I think something got broken here.

Previously the snapshot was rendering actual data, whilst the new snapshot is rendering mostly undefined errors.